### ISSUE TITLE
widget does not update 

### ISSUE THREAD

**Issue by lm4200** [justdave/nwsweatheralertswidget#33]
alerts expiring and getting a new alert  it says alert expired unable to view the alert on this app please update this app so it works better thank you.

**Comment by justdave** [comment]
Yeah, right now they don't disappear until the app polls the NWS server again (which happens every 5 minutes) and sees that the NWS doesn't have it posted anymore.  The expiration times are included in them, so I should indeed have it watch those times and remove it as soon as the time hits without waiting to poll the server again.

**Comment by lm4200** [comment]
i just view the alerts that in effect and it was a brand new alert issued
about 5 minutes ago  thank you for your help
